demos/showcase/showcase-webapp/src/main/webapp/WEB-INF/component/alloy/inputfile/header.xhtml

```diff
@@ -1,21 +1,2 @@
-<ui:composition xmlns="http://www.w3.org/1999/xhtml" xmlns:alloy="http://liferay.com/faces/alloy"
-	xmlns:c="http://xmlns.jcp.org/jsp/jstl/core" xmlns:f="http://xmlns.jcp.org/jsf/core"
-	xmlns:ui="http://xmlns.jcp.org/jsf/facelets" xmlns:showcase="http://xmlns.jcp.org/jsf/composite/showcase">
-
-	<showcase:note>
-		<h2>#{i18n['note']}</h2>
-		<ul>
-			<li><alloy:outputText escape="false" value="#{i18n['the-enclosing-form-must-have-multipart-form-data']} " /></li>
-			<li><alloy:outputLink value="http://ocpsoft.org/prettyfaces/" target="_blank">
-					<alloy:outputText value="PrettyFaces" />
-				</alloy:outputLink> <alloy:outputText escape="false" value="&amp;nbsp;" /> <alloy:outputText
-					value="#{i18n['has-compatibility-issues-with-jsf-22-file-upload-for-more-information-see-this']}" /> <alloy:outputText
-					escape="false" value="&amp;nbsp;" /> <alloy:outputLink
-					value="http://stackoverflow.com/questions/8047173/how-to-enable-multipart-form-data-in-tomcat-7-0-8-server/8050589#8050589"
-					target="_blank">
-					<alloy:outputText value="#{i18n['answer-at-stackoverflow']}" />
-				</alloy:outputLink></li>
-		</ul>
-	</showcase:note>
-
+<ui:composition xmlns="http://www.w3.org/1999/xhtml" xmlns:ui="http://java.sun.com/jsf/facelets">
 </ui:composition>
```

Comment:
@ngriffin7a, I removed the header warnings. I did this becuase I moved the one about `<form enctype="multipart/form-data">` down to the comments in the code. Also I don't think that it's really necessary to have a warning about prettyfaces [(which hasn't been updated since a year ago)](https://github.com/ocpsoft/prettyfaces/commits/prettyfaces3) in our showcase, (but if you disagree, I can put it back).